int32 controllerType
float64[] setpoint_null
float64 stiffness
float64 damping
